src/add-to-app/android/project-setup.md

```diff
@@ -41,64 +41,84 @@ android {
   the Flutter module still runs correctly.
 {{site.alert.end}}
 
-## Using Android Studio
+## Integrate your Flutter module
 
-The Android Studio IDE is a convenient way of integrating
-your Flutter module automatically. With Android Studio,
-you can co-edit both your Android code and your Flutter code
-in the same project. You can also continue to use your normal
-IntelliJ Flutter plugin functionalities such as Dart code
-completion, hot reload, and widget inspector.
+{% comment %} Nav tabs {% endcomment -%}
+<ul class="nav nav-tabs" id="add-to-app-android" role="tablist">
+    <li class="nav-item">
+        <a class="nav-link active" id="with-android-studio-tab" href="#with-android-studio" role="tab" aria-controls="with-android-studio" aria-selected="true">With Android Studio</a>
+    </li>
+    <li class="nav-item">
+        <a class="nav-link" id="without-android-studio-tab" href="#without-android-studio" role="tab" aria-controls="without-android-studio" aria-selected="false">Without Android Studio</a>
+    </li>
+</ul>
 
-Add-to-app flows with Android Studio are only supported on
-Android Studio 3.6 with version 42+ of the [Flutter plugin][]
-for IntelliJ. The Android Studio integration also only
-supports integrating using a source code Gradle subproject,
-rather than using AARs. See below for more details on
-the distinction.
+{% comment %} Tab panes {% endcomment -%}
+<div class="tab-content">
 
-Using the **File > New > New Module...** menu in
-Android Studio in your existing Android project,
-you can either create a new Flutter module to integrate,
-or select an existing Flutter module that was created previously.
+<div class="tab-pane active" id="with-android-studio" role="tabpanel" aria-labelledby="with-android-studio-tab" markdown="1">
 
-{% include docs/app-figure.md image="development/add-to-app/android/project-setup/ide-new-module.png" %}
+### Integrate with Android Studio
+{:.no_toc}
 
-If you create a new module, you can use a wizard to
-select the module name, location, and so on.
+The Android Studio IDE can help integrate your Flutter module.
+Using Android Studio, you can edit both your Android and Flutter code
+in the same IDE.
 
-{% include docs/app-figure.md image="development/add-to-app/android/project-setup/ide-wizard.png" %}
+You can also use your IntelliJ Flutter plugin functionality like
+Dart code completion, hot reload, and widget inspector.
 
-The Android Studio plugin automatically configures your
-Android project to add your Flutter module as a dependency,
-and your app is ready to build.
+Android Studio supports add-to-app flows on Android Studio 2022.1 or later
+with version 42 or later of the [Flutter plugin][] for IntelliJ.
```

Comment:
We can definitely drop this mention, since version 42 is from 2019 and only compatible with 2019.x versions of IntelliJ/AS.